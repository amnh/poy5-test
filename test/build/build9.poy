(* Build with constraint *)
(*terminals reduced to random 10 to expidite test*)
read ("23.fas")
select (terminals, _random:10)
set (seed:1)
build (4)
swap (tbr)
report (consensus)
build (constraint)