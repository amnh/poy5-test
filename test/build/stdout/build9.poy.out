Strict Majority Consensus Tree
(t35,t43,(t24,t26,t41))
